torch/_inductor/pattern_matcher.py

```diff
@@ -251,14 +251,73 @@ def replace_by_example(
             else contextlib.nullcontext()
         )
 
+        def should_propagate_arg_kwarg_vals(nodes: list[torch.fx.Node]) -> bool:
```

Comment:
is the a reason to not just have this take in a single node, if a list >= 1 will return False ?

Reply:
The check for >= 1 needs to exist somewhere, I put it into the function instead of outside of it. I can put it outside if you think that's clearer